### PR TITLE
Lower Power Mode Message

### PR DESCRIPTION
**Pull request by eudisd**
## Summary

As previously discussed, we will show a message informing the user that lower power mode is not supported.

Closes https://github.com/VIDY/embed.js/issues/143

## Screenshots

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -702,7 +702,17 @@ export default function Vidy(opts) {
...
-		if (iOS) video.play(); // iOS 11.4 last-resort attempt
+		if (iOS) {
+			video.play().catch(() => {
+				alert(
+					'Please make sure that your iPhone is not ' +
+					'in "Low Power Mode" in order to enable video autoplay' +
+					'\n\n(You\'ll need to reload the page once "Low Power Mode"' +
+					'is disabled.)'
+        );
+				resetPlayer();
```
Comment (lukeed) [inline src/index.js]: The `resetPlayer` has to be called first, as `alert` halts all execution until acknowledged

**Comment by lukeed** [comment]
Should we link to an official Apple post for instructions? Something like [this one](https://support.apple.com/en-us/HT205234)?

File: src/index.js (hunk 1 above)
Comment (eudisd) [reply src/index.js]: this still works as expected, but i'll move it up

**Comment by eudisd** [comment]
I think adding a like too it makes sense.  That way the user can look into it if they don't know how to turn off/on low power mode.  I'll add it now

**Comment by gwil** [comment]
`alert()` works for now but I'll improve UI during another sprint 👍

**Pull request merged**